couchbase/io/http_session.hxx

```diff
@@ -121,8 +121,8 @@ class http_session : public std::enable_shared_from_this<http_session>
       , ctx_(ctx)
       , resolver_(ctx_)
       , stream_(std::make_unique<plain_stream_impl>(ctx_))
```

Comment:
I think it would be better if the strand was owned by the http_session rather than the socket itself (and then having it shared outwards).  Though I don't think it's necessarily important enough to change right away.

Reply:
https://issues.couchbase.com/browse/CXXCBC-73